app/models/physical_switch.rb

```diff
@@ -40,8 +40,14 @@ def refresh_ems
     EmsRefresh.queue_refresh(ext_management_system)
   end
 
-  def event_where_clause(assoc = :ems_events)
-    ["#{events_table_name(assoc)}.physical_switch_id = ?", id]
+  # Must define these methods because EventMixin#belongs_to_column_id uses base class
+  # to guess the column name, which results in: switch_id.
```

Comment:
Instead, can this just override `belongs_to_column_id`?

Reply:
does feel like this wants to be an association